apps/include/apps.h

```diff
@@ -63,8 +63,9 @@ BIO *bio_open_owner(const char *filename, int format, int private);
 BIO *bio_open_default(const char *filename, char mode, int format);
 BIO *bio_open_default_quiet(const char *filename, char mode, int format);
 CONF *app_load_config_bio(BIO *in, const char *filename);
-CONF *app_load_config(const char *filename);
-CONF *app_load_config_quiet(const char *filename);
+CONF *app_load_config(const char *filename, int verbose);
+CONF *app_load_config_nonverbose(const char *filename, int quiet);
+#define app_load_config_quiet(filename) app_load_config_nonverbose(filename, 1)
```

Comment:
These 3 functions are confusingly named. What about having app_load_config, app_load_config_verbose and app_load_config_quiet function? This way it is somewhat more clear what is their purpose. Or have just one app_load_config function with verbosity argument where 0 is quiet, 1 is normal, 2 is verbose. Then of course you'll also need the constants #defined :smile:

Reply:
I also found your comment a bit confusing :wink: but I assume you meant the following:
Rename 
app_load_config, app_load_config_nonverbose, and app_load_config_quiet 
to 
app_load_config_verbose, app_load_config, and app_load_config_quiet, respectively.
Done.